src/ray/raylet/node_manager.cc

```diff
@@ -458,6 +462,8 @@ void NodeManager::ReportResourceUsage() {
   resources_data->set_node_id(self_node_id_.Binary());
 
   if (new_scheduler_enabled_) {
+    new_resource_scheduler_->UpdateLastReportResourcesFromGcs(
```

Comment:
Can you comment what this is for?

Reply:
this is for update cache in `new_resource_scheduler_ ` from gcs cache, because these two cache is different when gcs restart, I will add comment here.